src/main/java/org/benetech/servicenet/adapter/firstprovider/FirstProviderDataMapper.java

```diff
@@ -65,6 +84,21 @@ default Service extractService(RawData rawData) {
         return Arrays.stream(langs).map(lang -> new Language().language(lang)).collect(Collectors.toSet());
     }
 
+    default Set<OpeningHours> extractOpeningHours(RawData rawData) {
+        String[] rawOpeningHours = rawData.getWeeklyOpenHoursRaw().split(", ");
+        Set<OpeningHours> result = new HashSet<>();
```

Comment:
I think we might want to persist the order - any reason for using a set?

Reply:
Right